client.py

```diff
@@ -1 +1,47 @@
-# I'm empty!
+#!/usr/bin/python
+import sys, re, http.client, urllib.parse
+
+def main():
+    args = sys.argv[1:]
+    if len(args) != 4:
+        print ("ERROR: Invalid number of arguments")
+        return 1
+    
+    ip = re.match(IP_PATTERN, args[0], flags=0)
+    if ip is None:
+        print ("ERROR: Invalid IP address")
+        return 1
+    
+    port = re.match(PORT_PATTERN, args[1], flags=0)
+    if port is None:
+        print ("ERROR: Invalid port")
+        return 1
+
+    coords = re.match(COORDINATE_PATTERN, args[2] + " " + args[3], flags=0)
+    print (coords.group())
+    if coords is None:
+        print ("ERROR: Invalid Coordinates")
+        return 1
+
+    fire(ip.group(), port.group(), coords.group())
+
+def fire(ip, port, coords):
+    parameters = urllib.parse.urlencode({"x" : coords[0:1], "y" : coords[-1:]})
+    # For testing
+    print (parameters)
+
+    #This may not be quite correct
+    conn = http.client.HTTPConnection(ip, port)
+    conn.request("POST", parameters)
```

Comment:
I am looking into the `requests` [library](http://docs.python-requests.org/en/master/user/quickstart/) for this. Since you mention it might not be correct, take a look at it while I try to figure things out on my end.

Reply:
It works if you change it to connect to a server that's functional, we just have no way of making sure it runs for our server since we haven't set it up yet. The base code isn't an issue it's just making sure that it is accepted by our server